skins/oasis/modules/templates/Oasis_Index.php

```diff
@@ -68,7 +68,6 @@
 
 <?= $comScore ?>
 <?= $quantServe ?>
-<?= $googleAnalytics ?>
```

Comment:
This variable got cleaned up from the backend in May:

https://github.com/Wikia/app/commit/30a5a69b218294586312107aed8996b6b20e6cc7